test/unit/test_detailed_response.rb

```diff
@@ -20,7 +24,7 @@ def test_detailed_response_when_no_arguments
   end
 
   def test_detailed_response_when_only_response
-    response = IBMCloudSdkCore::DetailedResponse.new(response: HTTP::Response.new(status: 401, version: 1, body: {}))
+    response = IBMCloudSdkCore::DetailedResponse.new(response: HTTP::Response.new({ request: example_request, status: 401, version: 1, body: {} }))
```

Comment:
Thanks @pyrooka.  Good find.  I forgot to mention that the rubocop version is very old and isn't aware that these are no longer redundant with ruby 3.0+.  Keyword and positional arguments are different as of that version.  I kept it the way it would work with both ruby 2.7 and 3.0+.

The solution is to upgrade all of the test gems like rubocop including updating it's linter rules.  I hope to try to do that this week.  Hopefully, as a separate PR.

Reply:
FYI, we had to do many changes to get 3.0 support.  See https://github.com/ManageIQ/manageiq/pull/21531 for examples.  The high level is you get argument errors if you were passing hashes to methods expecting keyword argument errors and vice versa.  They are not the same thing anymore.

Reply:
Got it, thanks for the explanation! In this case I merge the PR and you can open a follow-up one with the updates you mentioned.